ci-operator/step-registry/ipi/conf/vsphere/zones/multisubnets/ipi-conf-vsphere-zones-multisubnets-commands.sh

```diff
@@ -99,6 +105,17 @@ networking:
   machineNetwork:
   - cidr: "${machine_cidr}"
 EOF
+
+# if loadbalancer is UserManaged, it's mean using external LB,
+# then keepalived and haproxy will not deployed, but coredns still keep
```

Comment:
What is difference compared with the existing `cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb` install workflow ?

Reply:
I also talk to Dev for this question, see https://issues.redhat.com/browse/OPNET-305?focusedId=23430205&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23430205

So like if using loadbalancer is UserManaged is IPI, if apiVIP and ingressVIP is empty, it's UPI

Reply:
okay. got it.